app/assets/images/icons/solid/arrow_long_right.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
+  <path fill-rule="evenodd" d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z" clip-rule="evenodd" />
+</svg>
```

Comment:
I think this is from Heroicons v2, right? I'd like to keep everything on [v1](https://v1.heroicons.com) for now and update them all at once.

Reply:
Right on. I ran into this on a personal project and it ended up being too late to fix 😂 Definitely better to stay on top of it. Good catch